content/libraries/multiformats/_index.md

```diff
@@ -1,22 +1,30 @@
 ---
 title: Multiformats
-description: Multiformats - self describing protocol values
-weight: 6
+weight: 4
 dashboardWeight: 1
-dashboardState: missing
-dashboardAudit: n/a
+dashboardState: stable
+dashboardAudit: missing
 dashboardTests: 0
 ---
 
-# Multiformats
+# Multiformats - self describing protocol values
 
-Self-describing protocol values
+[Multiformats](https://multiformats.io/) is a set of formating rules and formatting structures for self-describing values. These values are useful both to the data layer (IPLD) and to the network layer (libp2p). Multiformats includes specifications for the Content Identifier (CID) used by IPLD and IPFS, the multicodec, multibase and multiaddress (used by libp2p).
+
+Please refer to the [Multiformats repository](https://github.com/multiformats) for more information.
+
+## CIDs - Content IDentifiers
+
+Filecoin references data using IPLD's Content Identifier (CID).
+A CID is effectively a hash value, prefixed with its hash function (multihash) as well as extra labels, such as a `codec` and a `multibase` to inform applications about how to deserialize the given data.
```

Comment:
`multibase` isn’t part of CID, and I’m pretty sure we don’t use multibase in Filecoin for anything other than printing the CIDs.

I would maybe pair this down to something like:

“
A CID is a hash digest prefixed with identifiers for its hash function and codec. This means you can validate and decode data by with only this identifier.

When CIDs are printed as strings they also use `multibase` to identify the base encoding being used.
“